pdb-alignment-api/src/main/java/org/cbioportal/pdb_annotation/web/models/Ensembl.java

```diff
@@ -0,0 +1,65 @@
+package org.cbioportal.pdb_annotation.web.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "ensembl_entry")
+public class Ensembl {
+	// ------------------------
+	// PRIVATE FIELDS
+	// ------------------------
+	@Id 
+	@Column(name = "ENSEMBL_ID") 
+	private String ensemblid;
```

Comment:
Naming variables with camel case is the standard practice for Java. So, it might be better to name variables as `ensemblId`, `ensemblGene`, `ensemblTranscript`, etc.